pkg/adscert/signatory/signatory_local_impl.go

```diff
@@ -95,8 +95,6 @@ func (s *LocalAuthenticatedConnectionsSignatory) signSingleMessage(request *api.
 	sigInfo := &api.SignatureInfo{}
 	acs, err := formats.NewAuthenticatedConnectionSignature(formats.StatusOK, s.originCallsign, request.RequestInfo.InvokingDomain)
 	if err != nil {
-		acs.SetStatus(formats.StatusErrorOnSignature)
```

Comment:
Now that I look at this a bit more, I think the right solution here may be to update the NewAuthenticatedConnectionSignature function to return a partially populated struct rather than returning nil.  The original intent was to pass along whatever information could be populated, although that didn't get implemented quite right here.  I've been gradually adding tests to catch cases like this.

Reply:
yea make sense